feature-libs/product-configurator/rulebased/components/attribute/types/drop-down/configurator-attribute-drop-down.component.html

```diff
@@ -24,12 +24,12 @@
   >
     <option
       *ngFor="let item of attribute.values"
-      [label]="getLabel(expMode, item.valueDisplay, item.valueCode)"
       [selected]="item.selected"
+      [label]="getLabel(expMode, item.valueDisplay, item.valueCode, item)"
       [attr.aria-label]="getAriaLabel(item, attribute)"
       [value]="item.valueCode"
     >
-      {{ getLabel(expMode, item.valueDisplay, item.valueCode) }}
+      {{ getLabel(expMode, item.valueDisplay, item.valueCode, item) }}
```

Comment:
There is an issue with a dropdown list. For `Please select` there is a price:
![image](https://user-images.githubusercontent.com/61147963/213461362-b9c8c121-778d-49f1-9805-6cec7923a7c1.png)